src/services/database.ts

```diff
@@ -0,0 +1,14 @@
+import { DatabasePool, DatabaseTransactionConnection } from 'slonik';
+
+export declare type DatabasePoolHandler = DatabasePool;
+export declare type DatabaseTransactionHandler = DatabaseTransactionConnection;
+
+export interface Database {
```

Comment:
At some point we should probably suffix all services with `Service` (e.g. this is the convention in Symfony).